From 494e4abb7a5130376d3cdb41a5e3963650b8659f Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Mon, 16 Jul 2018 06:34:53 +0200
Subject: [PATCH] Fix the broken build due to python2

/builddir/build/BUILD/webkitgtk-2.23.3/Tools/gtkdoc/generate-gtkdoc
/usr/bin/env: 'python': No such file or directory
---
 Tools/gtk/generate-gtkdoc                                      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Tools/gtk/generate-gtkdoc b/Tools/gtk/generate-gtkdoc
index 45c23dfb..a3f82984 100755
--- a/Tools/gtkdoc/generate-gtkdoc
+++ b/Tools/gtkdoc/generate-gtkdoc
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # Copyright (C) 2011 Igalia S.L.
 #
 # This library is free software; you can redistribute it and/or
-- 
2.17.1

